Modules/xASL_module_ASL.m

```diff
@@ -566,6 +566,8 @@
         warning(['Missing field x.Q.' parNames{iPar} ', this may be needed for quantification']);
         % After the warning, we default to single-parameter processing
         fprintf('%s\n', ['Defaulting to single-' parAbbreviation{iPar} ' ASL processing']);
+		x.Q.(parNames{iPar}) = [];
```

Comment:
This should be illegal, every ASL flavor should have EchoTime, PLD, LD, because we will use them later in quantification. So we might as well make this an error instead of warning?